src/workflows/runCRISPRCasFinderl.cwl

```diff
@@ -26,11 +26,10 @@ inputs:
   spacer_fasta:
     type: string?
     label: Fasta with spacer sequences
-    default: "CRISPRCasFinder_Spacers.fasta"
   crisprcas_gff:
     type: string?
     label: GFF3 with CRISPRCas results
-    default: "CRISPRCasFinder.gff3"
```

Comment:
if there are no defaults, the value should be enforced by removing "?" from the type definition